tests/dataSources_test.go

```diff
@@ -803,15 +803,41 @@ var _ = Describe("DataSources", func() {
 				waitUntilDeployed()
 			})
 
-			It("[test_id:7469] should create DataImportCron", func() {
-				Expect(apiClient.Get(ctx, dataImportCron.GetKey(), dataImportCron.NewResource())).To(Succeed(), "custom DataImportCron not created")
+			It("[test_id:7469] should create DataImportCron in golden images namespace", func() {
+				Expect(apiClient.Get(ctx, dataImportCron.GetKey(), dataImportCron.NewResource())).To(Succeed(), "custom DataImportCron created")
+			})
+
+			It("[test_id:????] should create DataImportCron in other namespace", func() {
+				cronTemplate.Namespace = "ssp-operator-functests"
```

Comment:
The namespace `ssp-operator-functests` may not exist if the functests are run with an existing SSP installation.
Can you create a temporary namespace only for these tests?